src/client.rs

```diff
@@ -99,6 +99,12 @@ pub struct Client {
     connection: Connection,
 }
 
+impl Debug for Client {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Client")
```

Comment:
A better option here could be to use [`fmt::DebugStruct`](https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html):

```suggestion
        f.debug_struct("Client").finish()
```